pkg/scheduler/framework/plugins/podtopologyspread/filtering.go

```diff
@@ -280,6 +280,78 @@ func calPreFilterState(pod *v1.Pod, allNodes []*schedulernodeinfo.NodeInfo) (*pr
 	return &s, nil
 }
 
+// fit checks if pod is a good fit on the given node.
+func (pl *PodTopologySpread) fit(pod *v1.Pod, nodeInfo *nodeinfo.NodeInfo) *framework.Status {
```

Comment:
What is the benefit of this versus calling `calPreFilterState`? At a quick glance, complexity looks the same.